src/Symfony/Component/Finder/Tests/GitignoreTest.php

```diff
@@ -113,6 +113,21 @@ public function provider()
                 ['app/cache/file.txt', 'app/cache/subdir/ile.txt'],
                 ['a/app/cache/file.txt'],
             ],
+            [
+                '
+                /app/cache/
+                #LastLineIsComment',
```

Comment:
Can we also add some files after the comment?

Reply:
Yes, I'll add a new entry in the provider. Because I think the case where the last line is a comment, should be tested specifically. See also https://github.com/symfony/symfony/issues/32985#issuecomment-518653139